static/js/subs.js

```diff
@@ -727,6 +727,43 @@ $(function () {
         update_announce_stream_state();
     });
 
+    // Search People
+    $(document).on('input', '.add-user-list-filter', function (e) {
+        var users = people.get_rest_of_realm();
+        var i;
```

Comment:
You should put the loop variable inside of the loop clause (like `for (var i = 0; ...`)